blockly-ws-search/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "author": "Blockly Team",
+  "name": "@blockly/plugin-workspace-search",
+  "version": "1.20200304.0",
+  "description": "Implementation of workspace search for Blockly.",
+  "keywords": [
+    "blockly",
+    "workspace",
+    "search"
+  ],
+  "homepage": "https://github.com/google/blockly-samples/tree/workspace_search//blockly-ws-search#readme",
+  "bugs": {
+    "url": "https://github.com/google/blockly-samples/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/google/blockly-samples.git",
+    "directory": "blockly-ws-search"
+  },
+  "license": "Apache-2.0",
+  "main": "./dist/workspace-search.umd.js",
+  "unpkg": "./dist/workspace-search.umd.js",
+  "directories": {
+    "dist": "dist",
+    "src": "src"
+  },
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "clean": "rm -rf $npm_package_directories_dist",
+    "build": "webpack --display-error-details --env.mode=development",
+    "prepublishOnly": "npm run clean && webpack --display-error-details --env.mode=production",
+    "test": "webpack-dev-server"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.8.6",
+    "@babel/preset-env": "^7.8.6",
+    "babel-loader": "^8.0.6",
+    "blockly": "^3.20200123.1",
+    "webpack": "^4.41.6",
+    "webpack-cli": "^3.3.11",
+    "webpack-dev-server": "^3.10.3"
+  },
+  "peerDependencies": {
+    "blockly": ">=3.20200123.0 <4"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://wombat-dressing-room.appspot.com"
```

Comment:
Why add this here?

Reply:
It's recommended in docs so you don't have to add to the command line every time you want to publish.

Reply:
Awesome!